spaceship.zsh

```diff
@@ -33,29 +33,29 @@ SPACESHIP_GIT_UNPUSHED="${SPACESHIP_GIT_UNPUSHED:-⇡}"
 # NVM
 SPACESHIP_NVM_SHOW="${SPACESHIP_NVM_SHOW:-true}"
 SPACESHIP_NVM_SYMBOL="${SPACESHIP_NVM_SYMBOL:-⬢}"
-SPACESHIP_NVM_PREFIX="${SPACESHIP_NVM_PREFIX:-${SPACESHIP_PREFIX_ENV_DEFAULT}}"
 
 # RUBY
 SPACESHIP_RUBY_SHOW="${SPACESHIP_RUBY_SHOW:-true}"
 SPACESHIP_RUBY_SYMBOL="${SPACESHIP_RUBY_SYMBOL:-💎}"
-SPACESHIP_RUBY_PREFIX="${SPACESHIP_RUBY_PREFIX:-${SPACESHIP_PREFIX_ENV_DEFAULT}}"
-
-### XCODE
-SPACESHIP_XCODE_PREFIX="${SPACESHIP_XCODE_PREFIX:-${SPACESHIP_PREFIX_ENV_DEFAULT}}"
-
-### SWIFT
-SPACESHIP_SWIFT_PREFIX="${SPACESHIP_SWIFT_PREFIX:-${SPACESHIP_PREFIX_ENV_DEFAULT}}"
-
 
 # VENV
 SPACESHIP_VENV_SHOW="${SPACESHIP_VENV_SHOW:-true}"
-SPACESHIP_VENV_PREFIX="${SPACESHIP_VENV_PREFIX:-${SPACESHIP_PREFIX_ENV_DEFAULT}}"
 
 # VI_MODE
 SPACESHIP_VI_MODE_SHOW="${SPACESHIP_VI_MODE_SHOW:-true}"
 SPACESHIP_VI_MODE_INSERT="${SPACESHIP_VI_MODE_INSERT:-[I]}"
 SPACESHIP_VI_MODE_NORMAL="${SPACESHIP_VI_MODE_NORMAL:-[N]}"
 
+# Prefixes
+# Initializes prefix variables
+spaceship_prefixes() {
```

Comment:
Which looks like it breaks the explicit sets. 

I'm not sure how to get around this. The best way, maybe, is to take out the default initialization variable and set them all explicitly.